gradle/libs.versions.toml

```diff
@@ -3,15 +3,15 @@ kotlin = "1.7.0"
 save-core = "0.4.0-SNAPSHOT"
 ktor = "2.0.2"
 okio = "3.1.0"
-serialization = "1.3.2"
-kotlinx-datetime = "0.3.2"
-kotlinx-coroutines = "1.6.1"
-kotlin-wrappers = "0.0.1-pre.335"
+serialization = "1.3.3"
+kotlinx-datetime = "0.3.3"
+kotlinx-coroutines = "1.6.2"
+kotlin-wrappers = "1.0.0-pre.345"
 spring = "5.3.19"
-spring-boot = "2.6.7"
-spring-security = "5.6.3"
-spring-cloud = "3.1.2"
-spring-cloud-kubernetes = "2.1.2"
+spring-boot = "2.7.0"
+spring-security = "5.7.1"
```

Comment:
maybe add only spring-boot-dependencies (bom) instead of all spring dependencies?

Reply:
Tried to add it in https://github.com/saveourtool/save-cloud/pull/844/commits/ac7f5d83c294a9f2c9d0ea5a60ce66e8bddf5b55, please take a look.